src/components/hint/index.tsx

```diff
@@ -392,6 +407,11 @@ class Hint extends Component<HintProps, HintState> {
     return tipPositionStyle;
   }
 
+  shouldEnableShadow() {
+    const {color, enableShadow} = this.props;
+    return enableShadow && color === Colors.white;
```

Comment:
Why `enableShadow` works only when the color is white?
If so, it's counterintuitive to pass `color={white}` + `enableShadow` to make the shadow work

Reply:
Shadow works for a white container. If we want to support shadow in all cases we need to wrap the hint with another container. Since the request was for a shadow on a white bg hint I "saved" us a view on the hierarchy. I think we can add it if we'll have the requirement.

Reply:
No sure if we need to wrap the hint
If you just remove the `&& color === Colors.white`, (and `enableShadow={true}`) it will work with other colors 🙂

Reply:
Fixed ❤️